R/stats.R

```diff
@@ -45,11 +45,13 @@ sorted_count <- function(x) {
 #' @export
 
 inline_hist <- function(x) {
-  # Handle empty and NA vectors
-  if (length(x) < 1|| all(is.na(x))) 
+  # Handle empty and NA vectors (is.finite is FALSE for NA, NaN, and +/-Inf)
+  if (length(x) < 1|| !any(is.finite(x)))
```

Comment:
change this to all instead of !any.

Before the conditional  add something like 
`x[x==Inf| x == -Inf] <- NA`

(must be before the conditional in case it makes all the values NA)